model/Service/ConcurringSessionService.php

```diff
@@ -67,15 +69,19 @@ public function __construct(
         DeliveryExecutionService $deliveryExecutionService,
         FeatureFlagCheckerInterface $featureFlagChecker,
         StateServiceInterface $stateService,
+        TestSessionService $testSessionService,
```

Comment:
- [ ] Can you please make sure there is no other DI Service provider instantiating this class apart of `TestQtiServiceProvider`? Thanks!

Reply:
I've just checked and it is only used in the `TestQtiServiceProvider` from this extension, and all instantiations at runtime are done though the PSR container.

Reply:
![image](https://github.com/oat-sa/extension-tao-testqti/assets/339792/4869ea16-4f85-4b54-82c7-5c5ed47654be)